### DISCUSSION TITLE
[Feature] New widget: CRUD

### DISCUSSION THREAD

**Discussion by tabacitu** [Laravel-Backpack/community-forum#338] [Ideas]
A stripped-down version of a CrudController could be shown as a widget.

This could be used:
- in the dashboard, for fast access to something that's often-used;
- inside the Preview operation, to allow people to quickly do things to connected entries;

Food for thought. Let's brainstorm.

**Comment by ziming** [comment]
I think will be very useful, especially if a single dashboard page can show multiple tables & charts like an overview

- Latest Users
- Latest Orders
- Top Orders This week
- VIP Users this month

etc & u can perform actions on them
